avogadro/qtplugins/atompairbonds/atompairbonds.h

```diff
@@ -0,0 +1,55 @@
+/******************************************************************************
+  This source file is part of the Avogadro project.
+  This source code is released under the 3-Clause BSD License, (see "LICENSE").
+******************************************************************************/
+
+#ifndef AVOGADRO_QTPLUGINS_ATOMPAIRBONDS_H
+#define AVOGADRO_QTPLUGINS_ATOMPAIRBONDS_H
+
+#include <avogadro/qtgui/sceneplugin.h>
+
+namespace Avogadro {
+namespace QtPlugins {
+
+/**
+ * @brief Predict atom-pair interactions, like hydrogen bonds.
+ * @author Aritz Erkiaga
+ */
+class AtomPairBonds : public QtGui::ScenePlugin
+{
+  Q_OBJECT
+
+public:
+  explicit AtomPairBonds(QObject* parent = nullptr);
+  ~AtomPairBonds() override;
+
+  void process(const QtGui::Molecule& molecule,
+               Rendering::GroupNode& node) override;
+
+  QString name() const override { return tr(m_name.c_str()); }
```

Comment:
This doesn't work. You need to use the string here, or the translation scripts won't find it to add to the template.